kafka/kafka-docker.sh

```diff
@@ -16,6 +18,13 @@
 : ${KAFKA_LOG_CLEANER_ENABLE:=true}
 : ${KAFKA_ZOOKEEPER_CONNECT:=$ZOOKEEPER_PORT_2181_TCP_ADDR:$ZOOKEEPER_PORT_2181_TCP_PORT}
 : ${KAFKA_ZOOKEEPER_CONNECTION_TIMEOUT_MS:=6000}
+: ${KAFKA_AUTO_CREATE_TOPICS_ENABLE:=true}
+: ${KAFKA_DELETE_TOPIC_ENABLE:=true}
```

Comment:
I've remove the non-broker settings, thanks for that catch. My default settings which are opinionated based on our use case have been removed. You're quite right as settings can be done from the command line.